app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -57,47 +55,34 @@ class MainActivity : AppCompatActivity() {
 
         DatabaseManager.init(binding.root.context);
         CurrentProjectManager.loadFirstProject()
-
-
     }
 
     fun showMenuItems(items:Set<Int>){
-        allActionMenuIds.forEach{
-            this.Menu?.findItem(it)!!.isVisible = items.contains(it)
+        allActionMenuIds.forEach {
+            this.menu?.findItem(it)!!.isVisible = items.contains(it)
         }
     }
+
     fun setAppBarTitle(s:String){
         supportActionBar!!.title=s;
     }
 
-
     //todo this is probably not needed, res.menu.main.xml
     override fun onCreateOptionsMenu(menu: Menu): Boolean {
         // Inflate the menu; this adds items to the action bar if it is present.
         menuInflater.inflate(R.menu.main, menu)
-        this.Menu=menu;
+        this.menu=menu;
         onMenuInit?.menuHasInitialized();
-        makeMenuWhite(menu)
```

Comment:
I removed this method call because it was not helping your UI in any way. It was making the font of the drop down white...

I removed the function and the calling of it.